avm/res/api-management/service/README.md

```diff
@@ -45,12 +45,82 @@ The following section provides usage examples for the module, which were used to
 
 >**Note**: To reference the module, please use the following syntax `br/public:avm/res/api-management/service:<version>`.
 
-- [Using only defaults](#example-1-using-only-defaults)
-- [Using large parameter set](#example-2-using-large-parameter-set)
-- [Test deploying apim v2 sku](#example-3-test-deploying-apim-v2-sku)
-- [WAF-aligned](#example-4-waf-aligned)
+- [Using only defaults with a Consumption SKU](#example-1-using-only-defaults-with-a-consumption-sku)
+- [Using only defaults](#example-2-using-only-defaults)
+- [Using only defaults with a Developer SKU](#example-3-using-only-defaults-with-a-developer-sku)
+- [Using large parameter set](#example-4-using-large-parameter-set)
+- [Test deploying apim v2 sku](#example-5-test-deploying-apim-v2-sku)
```

Comment:
I'd suggest to update the corresponding title metadata to align with other sku tests, e.g., "Using a Basicv2 SKU"